src/libcommongui/commclient.cpp

```diff
@@ -198,7 +198,6 @@ void CommClient::onDisconnected() {
 
     QTcpSocket *socket = reinterpret_cast<QTcpSocket *>(sender());
     socket->deleteLater();
-    _tcpConnection = nullptr;
```

Comment:
Sure it's not useful?

Reply:
I had systematic crash on Linux in `CommClient::stop()`. I am guessing that the `_tcpConnection ` was set to `nullptr ` while being deleted in the other method.

Reply:
As discussed, reverted in a4083be53d1bf3223db2fb9883d5d5d58dcbb56a